classes/submission/form/ArticleGalleyForm.inc.php

```diff
@@ -86,7 +86,7 @@ function validate() {
 		$galleyDao =& DAORegistry::getDAO('ArticleGalleyDAO'); /* @var $galleylDao ArticleGalleyDAO */
 
 		$publicGalleyId = $this->getData('publicGalleyId');
-		$galleyWithPublicGalleyId = $galleyDao->getGalleyByPubId('publisher-id', $publicGalleyId, $this->articleId);
+		if ($publicGalleyId) $galleyWithPublicGalleyId = $galleyDao->getGalleyByPubId('publisher-id', $publicGalleyId, $this->articleId);
```

Comment:
Won't this lead to a potential uninitialized variable warning, since `$galleyWithPublicGalleyId` may not have been initialized?